src/mjolnir/shortcutbuilder.cc

```diff
@@ -736,8 +750,13 @@ void ShortcutBuilder::Build(const boost::property_tree::ptree& pt) {
   for (; tile_level != TileHierarchy::levels().rend(); ++tile_level) {
     // Create shortcuts on this level
     LOG_INFO("Creating shortcuts on level " + std::to_string(tile_level->level));
-    [[maybe_unused]] uint32_t count = FormShortcuts(reader, *tile_level);
-    LOG_INFO("Finished with " + std::to_string(count) + " shortcuts");
+    [[maybe_unused]] shortcut_stats stats = FormShortcuts(reader, *tile_level);
+    [[maybe_unused]] uint32_t avg = 0;
+    if (stats.first)
+      avg = stats.second / stats.first;
+    LOG_INFO("Finished with " + std::to_string(stats.first) + " shortcuts superseding " +
+             std::to_string(stats.second) + " edges, average ~" + std::to_string(avg) +
+             " edges per shortcut");
```

Comment:
print more info to the build logs. only the number of shortcuts is meaningless without the total number of superseded edges